Content.Server/Lightning/LightningSystem.cs

```diff
@@ -53,10 +54,14 @@ public void ShootLightning(EntityUid user, EntityUid target, string lightningPro
         var spriteState = LightningRandomizer();
         _beam.TryCreateBeam(user, target, lightningPrototype, spriteState);
 
-        var ev = new HitByLightningEvent(user, target);
-        RaiseLocalEvent(target, ref ev);
+        if (lightningPrototype != "Spark") // we don't want certain prototypes to trigger lightning level events
```

Comment:
Prototype comparisons, especially hardcoded ones, are a big "do not do" in this codebase. Not 100% on a better approach but this absolutely cannot be done.